jetstream/__init__.py

```diff
@@ -1,6 +1,8 @@
 import enum
 import re
 
+STATISTICS_SCHEMA_VERSION = 1
```

Comment:
Can we keep this next to the BigQuery schema definition in the StatisticsResult class?

Reply:
Absolutely!